pkg/ebpf/events_definitions.go

```diff
@@ -99,6 +103,10 @@ const (
 
 const Unique32BitSyscallsStartID = 3000
 
+const (
+	Ksymbols Requirement = iota
```

Comment:
When defining an enum, the convention is to put the type definition (`type Requirement uint8`) just beofre the consts